src/OpenTelemetry/OpenTelemetry.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFrameworks>$(TargetFrameworksForLibrariesExtended)</TargetFrameworks>
     <Description>OpenTelemetry .NET SDK</Description>
     <MinVerTagPrefix>core-</MinVerTagPrefix>
+    <!--<AnalysisLevel>latest-all</AnalysisLevel>-->
```

Comment:
soon :)

Reply:
Needed for detecting all stuff, I can fully remove this line if you prefer.
Still, there is ~100 more errors when it is enabled.

Reply:
Nono, leave it. I'm just happy to see it here. I understand this PR is a part 1. We're very close!